README.md

```diff
@@ -19,16 +19,16 @@ Eclipse Theia is an extensible framework to develop full-fledged multi-language
 
 </div>
 
-- [**Website**](#website)
```

Comment:
This was not an intended change. When I try to change it back locally, the stars `**` are automatically removed. I am guessing that using them is against some config we have somewhere. I think I would just leave out these stars, unless there are objections.